configs/teamcofh.json

```diff
@@ -5,13 +5,11 @@
   ],
   "stop_urls": [],
   "selectors": {
-    "lvl0": ".uk-container h1",
-    "lvl1": ".uk-container h2",
-    "lvl2": ".uk-container h3",
-    "lvl3": ".uk-container h4",
-    "lvl4": ".uk-container h5",
-    "text": ".uk-container p, .uk-container li"
+    "lvl0": "#cofh-docs-content h1",
+    "lvl1": "#cofh-docs-content h2",
+    "lvl2": "#cofh-docs-content h3",
+    "text": "#cofh-docs-content p, #cofh-docs-content li"
   },
-  "min_indexed_level": 1,
+  "min_indexed_level": 0,
```

Comment:
why did you change this?

Reply:
The search currently seems to only return results for subheaders (h2, h3, etc.). I would like it to return results for pages as a whole, by page title (h1). If I understand correctly, this option configures it so that higher level results (page titles) show up. Please correct me if I'm wrong though.